modules/react/toast/lib/Toast.tsx

```diff
@@ -34,11 +33,6 @@ export interface ToastProps {
    * The text of the Toast action.
    */
   actionText?: string;
-  /**
-   * The origin from which the Toast will animate.
-   * @default {horizontal: 'center', vertical: 'top'}
-   */
-  transformOrigin?: TransformOrigin;
```

Comment:
I feel like I'm missing something, but why is the `transformOrigin` prop being removed? This is still supported on `Popup.Card`

Reply:
The transform origin is determined by the `placement` in the `PopupModel`. This means the developer doesn't have to set this correctly, it will be handled by the chosen placement. This is an improvement for most popups since the Popup card will now animate away from the target. This effectively has no changes for Toast since they animate down by default and `bottom` is the default placement.

Reply:
The animation is also applied as a CSS style. It is still possible to override the animation by passing in a `css` prop or using `styled(Toast)` to override animations.